bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/helper/Setters.java

```diff
@@ -238,75 +244,93 @@ public static Resource setResource(Resource target, Resource source) {
         if (Objects.isNull(targetOnOff) && Objects.nonNull(sourceOnOff)) {
             target.setOnState(sourceOnOff);
         }
+
         // dimming
         Dimming targetDimming = target.getDimming();
         Dimming sourceDimming = source.getDimming();
         if (Objects.isNull(targetDimming) && Objects.nonNull(sourceDimming)) {
             target.setDimming(sourceDimming);
             targetDimming = target.getDimming();
+        } // minimum dimming level
```

Comment:
> Why was this comment moved?

No reason. I probably thought it looked pretty.

Reply:
It's a minor comment, but for me it makes the comment look like it belongs to the scope of lines 251-254, rather than the next lines, for which it's intended. Thus slightly confusing.

Reply:
Ok. Done.